### PR TITLE
Adding new problem

### PR DESCRIPTION
**Pull request by AlexSeagull**
For assignment 3 of SF-1

### CODE REVIEW

File: list-splitter/starter.py
Comment (Pana-stath) [inline list-splitter/starter.py:1]: extra stuff that maybe shouldn't be here(only the function should be present?) and the variables in the function are wrongly identified(was told if the type is a list no need to identify it and can't put int, str, etc in brackets).

**Review by Pana-stath — COMMENTED** [review]
last part might not be good but the rest looks fine. json validator says everything is good.

File: list-splitter/io.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  {
+    "input": [["no","way",3.14]],
+    "output": [["no","way"]]
+  },
+  {
+    "input": [[true,[1,2]]],
+    "output": [[]]
+  },
+  {
+    "input": [6,"nervous",26,"vulture",7],
```
Comment (emaj9) [inline list-splitter/io.json:19]: needs double [[ ]]

File: list-splitter/starter.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from typing import List
+
+def elementArranger(a: List[int,str,etc]) -> List[str]:
```
Comment (emaj9) [inline list-splitter/starter.py:3]: remove type annotation please

**Comment by emaj9** [comment]
Sorry I didn't catch that the outputs should NOT be in double square brackets

**Comment by emaj9** [comment]
@AlexSeagull see above

**Pull request merged**